minimal/Dockerfile.minimal

```diff
@@ -0,0 +1,7 @@
+FROM jasminegraph:squash
+
+ENV HOME="/home/ubuntu"
+ENV JASMINEGRAPH_HOME="${HOME}/software/jasminegraph"
+WORKDIR "${JASMINEGRAPH_HOME}"
+ENTRYPOINT ["/home/ubuntu/software/jasminegraph/run-docker.sh"]
+CMD ["ash"]
```

Comment:
@thevindu-w 
Is this `ash` or `bash`?

Reply:
It's `ash` because `bash` is not available in busybox docker image